src/etc/inc/auth.inc

```diff
@@ -488,7 +488,7 @@ function local_user_del($user)
 
 function local_user_set_password(&$user, $password)
 {
-    $user['password'] = crypt($password, '$6$');
+    $user['password'] = generate_password_hash($password, 10);
```

Comment:
The time is 2^cost_factor for bcrypt, so 10 is about 4 times faster than 12. And according to my research bcrypt with 10 is about 6 times longer than current impl with crypt/SHA512 (at my test box). (Although the cost factor has 'second importance' here, this pull request is about salting it the first place.)

Regarding the defaults, there might be none which can satisfy everyone, because the hashing speed varies between environments.

As above, the idea is not to bump the default in the sources, but to make it adjustable depending on the device it's running, so the admin can focus on _time_, not cost _factor_.
The intent of this push is more of a bugfix, not a feature. The feature should come in the future. ;-)